### ISSUE TITLE
Interesting project with similar goals: ParadeDB

### ISSUE THREAD

**Issue by data-envoy** [dekimir/postgres-searchbox#16]
Hey @dekimir if you haven't seen it already [ParadeDB](https://github.com/paradedb/paradedb) aims to simplify full-text (plus vector) search on Postgress.

There is cross-over with the projects goals, although I think they want to focus on backend and not front-end like InstantSearch.

I alse heard that Python then JS SDKs are on thier roadmap.

Something to keep in mind as this project goes forward.

**Comment by dekimir** [comment]
Yes, I saw it and mentioned postgres-searchbox to them [here](https://news.ycombinator.com/item?id=37814113).  I agree that this would be an awesome frontend for @ParadeDB.

**Comment by data-envoy** [comment]
Absolutely, I also mentioned this project to them on a call. So, if they wanted to go that route, it is on their radar.
